utils/auto_dockerfile_update/update_dockerfiles.py

```diff
@@ -9,6 +9,11 @@
 from get_dockerfiles import LAST_MODIFIED_REGEX
 from datetime import datetime, timezone
 
+AUTO_UPDATE_CONF_VERSION = ('python3', 'python3-deb')
+PYTHON3_REGEX = r'3\.\d+\.\d+'
+VERSION_CONF_REGEX = fr'version ?= ?{PYTHON3_REGEX}'
+BATCH_SIZE = 5
```

Comment:
if even 1 image fails it will prevent the others from merging. Ideally I would want no batches at all so only the ones that fail are stuck... each branch runs a new job concurrently in actions so I thought it might be a bit overkill to run each one seperately. WDYT?

Reply:
I see.
Then let's roughly divide to 3 batches?
See how many we had in the last few days to get a hunch.

Reply:
its trying right now to push out 4 batches